client/landing/stepper/declarative-flow/transferring-hosted-site-flow.ts

```diff
@@ -27,11 +31,29 @@ const transferringHostedSite: Flow = {
 	},
 	useStepNavigation( currentStep, navigate ) {
 		const siteId = useSiteIdParam();
-		const dispatch = useReduxDispatch();
-
+		const site = useSelect(
+			( select ) => ( siteId && ( select( SITE_STORE ) as SiteSelect ).getSite( siteId ) ) || null,
+			[ siteId ]
+		);
+		const wpcomAdminInterface = useSelector( ( state ) =>
```

Comment:
We can reuse

https://github.com/Automattic/wp-calypso/blob/a10dd9096313e84634a4fff71a99960ff9be9d30/client/state/sites/selectors/is-admin-interface-wp-admin.ts#L4

```js
const adminInterfaceIsWPAdmin = useSelector( ( state ) =>
	isAdminInterfaceWPAdmin( state, siteId )
);
```

Reply:
[Fixed here](https://github.com/Automattic/wp-calypso/pull/94644/commits/833c24020784feda5f9c6d7584c663ef624074e6), thanks!